plugins/default/lib/test_component_health.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+import unittest
+
+from component_health import parse_response
+
+class TestParseResponse(unittest.TestCase):
+    def runTest(self):
+        self.assertEqual(parse_response("{\"status\": \"ok\", \"summary\": \"Test Summary\", \"details\": []}"), (0, 'Test Summary', []))
+        self.assertEqual(parse_response("{\"status\": \"warn\", \"summary\": \"Test Summary\", \"details\": []}"), (1, 'Test Summary', []))
+        self.assertEqual(parse_response("{\"status\": \"crit\", \"summary\": \"Test Summary\", \"details\": []}"), (2, 'Test Summary', []))
```

Comment:
The exit status codes are important for nagios, so I would rather test for what we want explicitly in the test. We could of course use the nagios constants, but then you can change those values and not have the tests fail which I would prefer didn't happen.